packages/demo/pages/editable-html.js

```diff
@@ -17,7 +17,7 @@ import Typography from '@material-ui/core/Typography';
 import InputChooser from '../source/editable-html/input-chooser';
 import { hasText } from '@pie-lib/pie-toolbox/render-ui';
 // import { renderMath } from '@pie-lib/pie-toolbox/math-rendering-accessible';
-import { renderMath as renderMathOld } from '@pie-lib/pie-toolbox/math-rendering';
+import { renderMath } from '@pie-lib/pie-toolbox/math-rendering';
```

Comment:
Shouldn't `renderMath` be imported from `@pie-lib/pie-toolbox/math-rendering-accessible` instead?